src/index.html

```diff
@@ -19,9 +19,10 @@
   <!-- END Google Analytics Scripts -->
   <meta charset="utf-8">
   <title>CLARK | Cybersecurity Labs and Resource Knowledgebase</title>
-  <meta name="description" content="CLARK is a prototype curriculum management platform that will host diverse cybersecurity learning objects. It was created
-  because there is a demonstrated need for a high-quality and high-availability repository for curricular and ancillary resources
-  in the cybersecurity education community.">
+  <meta name="description" content="CLARK is a digital library that hosts diverse cybersecurity
+  learning objects. It was created because there is a demonstrated need for a high-quality and
+  high-availability repository for curricular and ancillary resources in the cybersecurity education
+  community. The curriculum in CLARK is free, resides in several collections, and undergoes stringent reviews..">
```

Comment:
Looks like an extra period was left in:

```suggestion
  community. The curriculum in CLARK is free, resides in several collections, and undergoes stringent reviews.">
```